src/app/components/Process.tsx

```diff
@@ -0,0 +1,47 @@
+import Image from 'next/image';
+import { useTranslations } from 'next-intl';
+
+import Animation from './Animation';
+
+import moon_group from '../../../public/process/moon_group.png';
+import astronaut from '../../../public/process/astronaut.png';
+
+const Process = () => {
+  const t = useTranslations('Process');
+
+  return (
+    <div className="font-body bg-black px-6 py-14 mx-auto mt-24 flex flex-col items-center text-center 2xl:mt-36 2xl:py-24">
+      <div>
+        <Animation delay={1}>
+          <h1 className="text-white text-4xl font-black md:text-4xl md:mx-36 xl:text-5xl 2xl:text-6xl">
+            {t('processHeaderTitle')}
+          </h1>
+          <Image
+            src={moon_group}
+            alt="moon image"
+            className="w-full mt-6 2xl:w-4/5 2xl:mx-auto"
+          />
+        </Animation>
+      </div>
+
+      <div className="mt-12 flex items-center gap-5 xl:w-3/4 2xl:w-2/4">
+        <Image
+          src={astronaut}
+          alt="moon image"
+          className="h-80 md:h-auto md:w-96"
```

Comment:
the image looks stretched on mobile breakpoint, maybe you should use a fixed value for different breakpoints?